TheMovieDb/Controller/ReviewsViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ReviewsViewController.swift
+//  TheMovieDb
+//
+//  Created by Karla Rubiano on 1/11/21.
+//
+
+import UIKit
+
+final class ReviewsViewController: UIViewController  {
+    
+    var movieID: Int?
+    var movie: Movie?
+    var reviews: [ReviewsDetails] = []
```

Comment:
Internal properties?

Reply:
Changed `var movie` and `var reviews` to private.